data/posts/about.markdown

```diff
@@ -1,15 +1,15 @@
 # Proč WeddingCon?
 
-Pořádáme konference (anglicky "conference") a navštěvujeme srazy fanoušků (anglicky "convention"). Tyto akce mají často v názvu koncovku "con". Svatbu prakticky bereme jako další konferenci, hodně věcí je podobných. Bude mít jenom do 100 účastníků, takže vlastně pohoda :) I tento web používá [šablonu na konference](https://github.com/gdg-x/hoverboard).
+Pořádáme konference (anglicky „conference“) a navštěvujeme srazy fanoušků (anglicky „convention“). Tyto akce mají často v názvu koncovku „con“. Svatbu prakticky bereme jako další konferenci, hodně organizačních věcí je podobných. Bude mít jenom do 100 účastníků, takže vlastně pohoda :) I tento web používá [šablonu na konference](https://github.com/gdg-x/hoverboard).
 
-Náš WeddingCon bude tradičně netradiční - máme rádi tradice, ale každou plánujeme opepřit něčím, co máme rádi: technologiemi, metalovou hudbou, cestováním a dalším.
+Náš WeddingCon bude tradičně netradiční – máme rádi tradice, ale každou plánujeme opepřit něčím, co máme rádi: technologiemi, metalovou hudbou, cestováním a dalším.
 
 # Jak jsme se poznali?
 
-Naše seznámení bylo hodně neobvyklé, jak z indického filmu. Bára má rodinnou známou ("tetičku"), která ráda dává lidi dohromady. Jednou šla v Černošicích do lékárny, kde potkala tátu. Padla řeč na to, jestli má svobodného syna. Tetička povídala tak dlouho, až táta souhlasil. Najednou Davidovi píše neznámá holka z Černošic. David jí odepsal, že jí nezná a tím to skončilo. Tetička šla znovu do lékárny, táta se omlouval už jak jí viděl a nakonec Davidovi zavolal. Oba souhlasili se schůzkou jenom "aby byl klid". Ale na schůzce v Café Alternatif jsme si proti všem očekáváním padli do oka.
+Naše seznámení bylo hodně neobvyklé, skoro jako z indického filmu. Bára má rodinnou známou („tetičku“), která ráda dává lidi dohromady. Jednou šla v Černošicích do lékárny, kde se zapovídala s Davidovým tátou. Padla řeč na to, že má svobodného syna. Tetička povídala tak dlouho, až táta souhlasil s předáním kontaktu. Najednou Davidovi píše neznámá holka z Černošic. David jí odepsal, že ji nezná, a tím to skončilo. Bára si stěžovala tetičce, ta zašla znovu do lékárny. Táta se omlouval hned, jak ji viděl, a nakonec Davidovi zavolal. Oba souhlasili se schůzkou jenom „aby byl klid“. Ale na schůzce v Café Alternatif si proti všem očekáváním padli do oka.
```

Comment:
To "oba" není jasný, může se to vztahovat na Davida a tátu. Asi bych přepsal na "David a Bára"